pubspec.lock

```diff
@@ -727,14 +720,14 @@ packages:
       name: url_launcher_android
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "6.0.15"
+    version: "6.0.14"
   url_launcher_ios:
     dependency: transitive
     description:
       name: url_launcher_ios
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "6.0.15"
+    version: "6.0.14"
```

Comment:
I'm curious why these transitive dependencies decreased in version as a result of removing `dropdown_plus`.

EDIT: Now looking at the individual commits, this originated before the change. Hmm, well I think anyhow we'll update pubspec.lock later on so this can be merged.

Reply:
Yeah am not entirely sure too - there were some things that I updated on my end that removed some of these changes but there still seems to be some config differences.